### PR TITLE
Document automate-ctl gather-logs --all-logs flag

### PR DESCRIPTION
**Pull request by danielsdeleo**
Also explain the criteria by which logs are selected by default.

Signed-off-by: Daniel DeLeo <dan@chef.io>

### CODE REVIEW

File: chef_master/source/ctl_automate_server.rst [hunk 1]
```diff
@@ -246,13 +246,16 @@ gather-logs
 =====================================================
 The ``gather-logs`` command is used to collect the logs from Chef Automate into a compressed file archive. Once it runs it will create a tbz2 file in the current working directory with the timestamp as the file name.
 
+By default, it collects the most current logfile as well as any others that have been modified in the last 180 minutes. If the ``--all-logs`` option is given, all log files are collected.
```
Comment (markan) [inline chef_master/source/ctl_automate_server.rst]: Do you think it's clear enough from context that it's only Automate logfiles?

**Review by markan — APPROVED** [review]
Other than minor comment, looks good.

File: chef_master/source/ctl_automate_server.rst (hunk 1 above)
Comment (danielsdeleo) [reply chef_master/source/ctl_automate_server.rst]: The first sentence says

> collect the logs from Chef Automate into a compressed file archive

so I think we're ok.

File: chef_master/source/ctl_automate_server.rst [hunk 2]
```diff
@@ -246,13 +246,16 @@ gather-logs
...
+
 **Syntax**
 
 This subcommand has the following syntax:
 
 .. code-block:: bash
 
    $ automate-ctl gather-logs
+        --all-logs  Grab all of the logs, without regard to size or age. WARNING: could get very large
```
Comment (brewn) [inline chef_master/source/ctl_automate_server.rst]: Should this example all be on one line? ie:
`automate-ctl gather-logs --all-logs`

**Comment by brewn** [comment]
tenor-132962945

**Pull request merged**